lib/src/forms/include_test_form.dart

```diff
@@ -0,0 +1,184 @@
+            ),
+          ],
+          minify: minify,
+        ),
+      ],
+    );
+  }
+}
+
+class _TestEditor extends StatefulWidget {
+  _TestEditor({@required this.values});
+
+  final Map<String, dynamic> values;
+
+  @override
+  _TestEditorState createState() => _TestEditorState();
+}
+
+class _TestEditorState extends State<_TestEditor> {
+  List<PendingTest> _availableTests;
+  TextEditingController _suiteController;
+  TextEditingController _testController;
+  Translator _translator;
+
+  @override
+  void initState() {
+    super.initState();
+    _initAvailableTests();
+    _initTextControllers();
+    _translator = Translator.of(context);
+  }
+
+  @override
+  void dispose() {
+    _suiteController.dispose();
+    _testController.dispose();
+    super.dispose();
+  }
+
+  Future<void> _initAvailableTests() async {
+    _availableTests = [];
+    var runner = TestRunner.of(context);
+    var tests = await runner.controller.loadTests(context);
+    tests.forEach(
+      (test) => _availableTests.add(test),
+    );
+  }
+
+  void _initTextControllers() {
+    _suiteController = TextEditingController();
+    _suiteController.text = widget.values['suiteName'];
+    _suiteController.addListener(() {
+      _updateValues(
+        widget.values,
+        suiteName: _suiteController.text,
+      );
+    });
+
+    _testController = TextEditingController();
+    _testController.text = widget.values['testName'];
+    _testController.addListener(() {
+      _updateValues(
+        widget.values,
+        testName: _testController.text,
+      );
+    });
+  }
+
+  List<PendingTest> _getSuggestionsFrom({
+    int limit = 4,
+    @required String testName,
+  }) {
+    var result = _availableTests
+        .where(
+          (test) {
+            return test.name.toLowerCase().contains(
+                  testName.toLowerCase(),
+                );
+          },
+        )
+        .take(limit)
+        .toList();
+    return result;
+  }
+
+  void _updateValues(
+    Map<String, dynamic> values, {
+    String suiteName,
+    String testName,
+  }) {
+    values['suiteName'] =
+        suiteName?.isNotEmpty != true ? values['suiteName'] : suiteName;
+    values['testName'] =
+        testName?.isNotEmpty != true ? values['testName'] : testName;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var translatedSuiteName = _translator.translate(
+      TestFlowControlTranslations.atf_flow_form_suite_name,
+    );
+    var translatedTestName = _translator.translate(
+      TestFlowControlTranslations.atf_flow_form_test_name,
+    );
+    return Column(
+      children: [
+        TextFormField(
+          controller: _suiteController,
+          decoration: InputDecoration(
+            labelText: translatedSuiteName,
+          ),
+        ),
+        TypeAheadFormField(
+          autovalidate: true,
+          hideOnEmpty: true,
+          itemBuilder: (context, test) {
+            return ListTile(
+              title: Text(
```

Comment:
Please sort the attributes alphabetically for consistency with the rest of the framework.